sdk/include/opentelemetry/sdk/trace/samplers/probability.h

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020, Open Telemetry Authors
+// Copyright 2017, OpenCensus Authors
```

Comment:
What part of this is taken from OpenCensus?

Reply:
Just the function headers. I wasn't sure if the copyright was needed in the header as well but I decided to err on the cautious side

Reply:
Which function headers did you take? `CalculateThreshold`? I'm not sure if we can have copyright notices like this. See my comment [here](https://github.com/open-telemetry/opentelemetry-cpp/pull/123/files#r443672385).

Reply:
I took `CalculateThreshold` and `CalculateThresholdFromBuffer`. Is this comment in reference to the copyright notices in both probability.h and probability.cc, or just probability.h? @maxgolov pointed out something similar that was done in [nostd/variant.h](https://github.com/open-telemetry/opentelemetry-cpp/blob/3bbd2806c4699c5dadff7148217ee3d2b1d576aa/api/include/opentelemetry/nostd/variant.h), but is there a different way that this should be done?

Reply:
I see. Let's be cautious and leave the copyright notice in.